script.js

```diff
@@ -26,4 +26,44 @@ scrollToLinks.forEach((link) => {
 });
 unionButton.addEventListener('click', showMobileMenu);
 
-crossButton.addEventListener('click', hideMobileMenu);
\ No newline at end of file
+crossButton.addEventListener('click', hideMobileMenu);
+
+document.addEventListener('DOMContentLoaded', () => {
+  const form = document.getElementById('form');
+  const username = document.getElementById('name');
+  const email = document.getElementById('email');
+
+  function setError(elm, msg) {
+    const inputControl = elm.parentElement;
+    const errorDiv = inputControl.querySelector('.error');
+    errorDiv.innerText = msg;
+    inputControl.classList.add('error');
+  }
+
+  function clearError(elm) {
+    const inputControl = elm.parentElement;
+    const errorDiv = inputControl.querySelector('.error');
+    errorDiv.innerText = '';
+    inputControl.classList.remove('error');
+  }
+
+  function validateFormInputs() {
+    const usernameValue = username.value.trim();
+    const emailValue = email.value.trim();
+
+    clearError(username);
+    clearError(email);
+
+    if (usernameValue === '') {
+      setError(username, 'Username is required');
+    }
+    if (emailValue === '') {
+      setError(email, 'Email is required');
+    }
+  }
+
+  form.addEventListener('submit', (event) => {
+    event.preventDefault();
+    validateFormInputs();
+  });
+});
```

Comment:
Please note that you are yet to implement all the requirements for this milestone. Kindly see the below screenshot for the missing requirement 👍🏽
![Screenshot from 2023-05-16 16-38-27](https://github.com/zuhebahmed88091/Portfolio_site/assets/44624138/6a751110-23ef-41c0-bf92-398a105dc222)